spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,144 @@
+require 'rails_helper'
+
+RSpec.describe 'Articles', type: :request do # rubocop:disable Metrics/BlockLength
+  headers = { 'Authorization': "Token token=#{ENV['HOWITZER_TOKEN']}" }
+
+  describe 'GET #index' do
+    let!(:articles) { create_list(:article, 10) }
+    before { get api_v1_articles_path, headers: headers }
+
+    it 'returns a successful response' do
+      expect(response).to be_successful
+    end
+
+    it 'returns the list of articles' do
+      expect(json.map { |x| x['id'] }).to eq [*1..10]
+    end
+  end
+
+  describe 'GET #show' do
+    let!(:articles) { create_list(:article, 5) }
+    context 'existing article' do
+      before { get '/api/v1/articles/3', headers: headers }
```

Comment:
Who will guarantee there will be 1, 2, 3, 4, 5 ids in db for articles. Bad and not stable solution. Some day your tests were blinking. I recommend to create 2 articles separately. and use #{article1.id}

Reply:
done